basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [Role-based authentication (RBAC)](/role-based-access-control.md)                                        | Y            | Y            | Y            | Y            | Y            | N            |
+| [Certificate-based authentication](/certificate-authentication.md)                                       | Y            | Y            | Y            | Y            | Y            | N            |
+| Support for MySQL 8.0 clients                                                                            | Y            | Y            | Y            | N            | N            | N            |
+| [MySQL compatible `GRANT` system](/privilege-management.md)                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Dynamic Privileges](/privilege-management.md#dynamic-privileges)                                        | Y            | N            | N            | N            | N            | N            |
+| [Security Enhanced Mode](/system-variables.md#tidb_enable_enhanced_security)                             | Y            | N            | N            | N            | N            | N            |
+| [Redacted Log Files](/log-redaction.md)                                                                  | Y            | Y            | N            | N            | N            | N            |
+| **Data import and export**                                                                               | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Fast Importer (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md)                             | Y            | Y            | Y            | Y            | Y            | Y            |
+| mydumper logical dumper                                                                                  | Deprecated   | Deprecated   | Deprecated   | Y            | Y            | Y            |
+| [Dumpling logical dumper](/dumpling-overview.md)                                                         | Y            | Y            | Y            | N            | N            | N            |
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                  | Y            | Y            | N            | N            | N            | N            |
+| [Database migration toolkit (DM)](/migration-overview.md)                                                | Y            | Y            | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                      | Deprecated   | Deprecated   | Deprecated   | Y            | Y            | Y            |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                    | Y            | Y            | Y            | N            | N            | N            |
```

Comment:
@IANTHEREAL Can you please review this section (Data import and export)?

Reply:
I think CDC is not GA on 4.0, maybe it should be from 4.0.6 or just mark it as "Experimental" in 4.0.

Reply:
I had exactly the same opinion with @morgo (but failed to leave the message earlier :( )